searx/engines/bing_videos.py

```diff
@@ -28,74 +32,87 @@
 number_of_results = 28
 
 base_url = 'https://www.bing.com/'
-search_string = 'videos/search'\
-    '?{query}'\
-    '&count={count}'\
-    '&first={first}'\
-    '&scope=video'\
+search_string = (
+    'videos/search'
+    '?{query}'
+    '&count={count}'
+    '&first={first}'
+    '&scope=video'
     '&FORM=QBLH'
+)
 time_range_string = '&qft=+filterui:videoage-lt{interval}'
-time_range_dict = {'day': '1440',
-                   'week': '10080',
-                   'month': '43200',
-                   'year': '525600'}
+time_range_dict = {
+    'day': '1440',
+    'week': '10080',
+    'month': '43200',
+    'year': '525600',
+}
 
 # safesearch definitions
-safesearch_types = {2: 'STRICT',
-                    1: 'DEMOTE',
-                    0: 'OFF'}
-
+safesearch_types = {
+    2: 'STRICT',
+    1: 'DEMOTE',
+    0: 'OFF',
+}
 
-# do search-request
 def request(query, params):
-    offset = ((params['pageno'] - 1) * number_of_results) + 1
 
+    offset = ((params['pageno'] - 1) * number_of_results) + 1
     search_path = search_string.format(
-        query=urlencode({'q': query}),
-        count=number_of_results,
-        first=offset)
+        query = urlencode({'q': query}),
+        count = number_of_results,
+        first = offset,
+    )
 
     # safesearch cookie
-    params['cookies']['SRCHHPGUSR'] = \
+    params['cookies']['SRCHHPGUSR'] = (
         'ADLT=' + safesearch_types.get(params['safesearch'], 'DEMOTE')
+    )
 
     # language cookie
-    language = match_language(params['language'], supported_languages, language_aliases).lower()
+    language = match_language(
+        params['language'], supported_languages, language_aliases
+    ).lower()
```

Comment:
I don't understand why the function is split into 3 lines.

Reply:
Just my subjective feeling of _readability_ / don't let us look to close, instead lets enforce https://github.com/searxng/searxng/pull/619 where we can define our Code-Style commitment.